cli/sdk/src/sendPanic.ts

```diff
@@ -9,6 +8,7 @@ import os from 'os'
 import path from 'path'
 import stripAnsi from 'strip-ansi'
 import tmp from 'tmp'
+import * as checkpoint from 'checkpoint-client'
```

Comment:
We can simplify this to `import checkpoint from 'checkpoint-client'` as we have `esModuleInterop: true` in our tsconfig.json.

Reply:
Indeed!